source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -288,13 +288,23 @@ Wallets
 |bw|
 -------------------------
 
-    April 4, 2024
+    June 20, 2024
 
-    Version 1.5.1 fixes a bug related to the injected script being loaded multiple times which was causing performance issues on the browser.
+    Version 1.5.2 includes several changes and bug fixes.
+        -   Added new option to edit account name. Name saved in local storage. Changed name displayed across all BrowserWallet.
```

Comment:
I suggest not mentioning `local storage`. It seems an internal implementation detail.

```suggestion
        -   Added a new option to edit account names. The set names are used across the browser wallet for referring to the accounts.

```